docs/capabilities.md

```diff
@@ -4,33 +4,100 @@ time.  Whether you are working with large legacy codebases, your own
+
+Atomist consumes events from the platforms and tools you use every day.
+By linking related events from source code repositories, continuous
+integration, testing, and runtime environments, Atomist can present
+them in a unified manner to appropriate team members via direct or
+group messages.  The result is massive reductions in context-switching
+costs and and shorter incident time-to-resolution.
+
+<!-- *That rug really tied the room together, did it not?* -->
+
+Rug is the eventing runtime that ties everything together.  Rug not
+only understands events, but can respond to events with appropriate,
+automated actions.
+
 ## Development lifecycle automation
-### Event-driven actions: handlers
-### ChatDev and ChatOps: commands
-### Project updates: editors
-### Project compliance: reviewers
-### Project creation: generators
-## Events and code selection: tree expressions
-## Interfaces
-### XX Button XX
-### Bot
-### CLI
+
+Atomist helps you develop better software more quickly by greatly
+enhancing your ability to automate your software development
+lifecycle.  By intelligently linking your source code repository, your
+CI system, your deployment platform, your logs, and your alerting
+system, Atomist connects code changes to runtime changes and provides
+that information back, with its full context, back to the appropriate
+team member.  It can can even automate remediation, obviating the need
+to alert anyone.
+
+The Rug programming model provides several different types of Rugs to
+help you manage your development and operations lifecycle.
+
+<!-- can we improve the layout of the items below? -->
+
+**Event-driven actions: <span class="rugs">[Handlers][handlers]</span>** - Rug
+handlers respond to events.  Handlers can use the information on an
+event and its related events to message the right team members, take
+direct action, and/or create new events, which can trigger other
+handlers.
+
+**ChatDev and ChatOps: <span class="rugs">[Commands][commands]</span>** - Rug
+commands are tasks you can execute within chat, automating common
+tasks in software development and operation.  Want to release a new
+version of your service to production?  Do it right in chat with a
+command.
+
+**Project updates: <span class="rugs">[Editors][editors]</span>** - Rug editors
+modify code directly and consistently.  Need to add or update a
+dependency and boilerplate code?  Codify it in an editor.
+
+**Project creation: <span class="rugs">[Generators][generators]</span>** - Rug
+generators create new projects from a model project.  More powerful
+than templating approaches, generators are real projects operating
+under native tooling.
+
+**Project compliance: <span class="rugs">[Reviewers][reviewers]</span>** - Rug
+reviewers ensure a project conforms with your standards.  Like an
+editor, without the editing.
+
+Individual Rugs can be composed.  For example, editors can be used by
+Rug handlers, commands, and generators to modify code in a consistent
```

Comment:
(I didn't try very hard to be neat, since a real design person will design them)